pkg/daemon/gpsd.go

```diff
@@ -24,6 +25,17 @@ const (
 	GNSSMONITOR_INTERVAL = 1 * time.Second
 )
 
+type filteringStderrWriter struct{}
+
+func (w *filteringStderrWriter) Write(p []byte) (n int, err error) {
+	if bytes.Contains(p, []byte("Inappropriate ioctl for device")) {
+		// Suppress this error
+		return len(p), nil
```

Comment:
Should this return 0? 
I'm not sure.

Reply:
Great question . I am acting like AI :-P 
n = number of bytes successfully “written”
err is nill , might make cmd.StdoutPipe() think nothing was return nor error

Reply:
Ah ok that makes sense thanks :)